protocol/amqp/v2/protocol.go

```diff
@@ -92,69 +101,76 @@ func NewProtocol(server, queue string, connOption []amqp.ConnOption, sessionOpti
 }
 
 // NewSenderProtocolFromClient creates a new amqp sender transport.
-func NewSenderProtocolFromClient(client *amqp.Client, session *amqp.Session, address string, opts ...Option) (*Protocol, error) {
+func NewSenderProtocolFromClient(
+	ctx context.Context,
+	client *amqp.Conn,
+	session *amqp.Session,
+	address string,
+	opts ...Option,
```

Comment:
seem to be unused?

Reply:
what do you mean? which property? opts are applied and sent in the sender creation